db/migrate/20150724182915_add_subject_and_comment_template_for_email_settings.rb

```diff
@@ -0,0 +1,24 @@
+class AddSubjectAndCommentTemplateForEmailSettings < ActiveRecord::Migration
```

Comment:
You mean the defaults? Yep, that could also be an option but first we should answer the questions I raised in my other comment ...